File: lib/api/models/beers.js

//mongoimport --db beer-list --collection beers --drop --jsonArray --file ./beers.json
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var BeerSchema = new Schema({  
    id: Number,
    brewery: String,
    beer: String,
    style: String,
    season: String,
    abv: Number,
    ibu: Number,
    description: String
}, 
    { collection : 'beers' });

module.exports = mongoose.model('Beer', BeerSchema);

File: app/models/beers.rb
class Beers
  include Mongoid::Document
  field :id, type: Integer
  field :brewery, type: String
  field :beer, type: String
  field :style, type: String
  field :season, type: String
  field :abv, type: Float
  field :ibu, type: Float
  field :description, type: String
end
